custom/icds_reports/views.py

```diff
@@ -430,6 +430,58 @@ def get_state_id_from_state_name(self, state_name):
         return SQLLocation.objects.get(name=state_name, location_type__name='state').location_id
 
 
+@location_safe
+@method_decorator([login_and_domain_required,
+                   toggles.ENABLE_ICDS_DASHBOARD_RELEASE_NOTES_UPDATE.required_decorator(),
+                   require_superuser], name='dispatch')
+class ReleaseNotesUpdateView(TemplateView):
+    page_title = 'Update Dashboard Release Notes'
+    urlname = 'update_dashboard_release_notes'
+    template_name = 'icds_reports/update_release_notes.html'
+
+    def post(self, request, *args, **kwargs):
+        data = request.FILES['dashboard_release_notes']
+        icds_file, _ = IcdsFile.objects.get_or_create(blob_id="dashboard_release_notes.pdf",
+                                                      data_type='dashboard_release_notes')
+
+        icds_file.store_file_in_blobdb(data)
+        icds_file.save()
+        messages.success(request, 'ICDS Dashboard Release Notes uploaded Successfully')
+        return redirect(self.urlname, domain=request.domain)
+
+
+@location_safe
+@method_decorator([login_and_domain_required], name='dispatch')
+class DownloadReleaseNotes(View):
+    def get(self, request, *args, **kwargs):
+        release_notes_file = IcdsFile.objects.filter(blob_id="dashboard_release_notes.pdf")\
```

Comment:
good to also put the `data_type` in the filter, it will improve the query time as it will have to search lesser number of records for that blob_id